ipfs/ipfs_test.go

```diff
@@ -1,46 +1,63 @@
 package ipfs_test
 
-//type testdata struct {
-//	Name        string `json:"name"`
-//	Description string `json:"description"`
-//}
-//
-//const (
-//	ipfsNodeAddr = "ipfs.io/ipfs"
-//)
-//
-//func TestIpfsAdd(t *testing.T) {
-//	testData := &testdata{
-//		Name:        "panacea",
-//		Description: "medibloc mainnet",
-//	}
-//
-//	newIpfs := store.NewIpfs(ipfsNodeAddr)
-//
-//	testDataBz, err := json.Marshal(testData)
-//	require.NoError(t, err)
-//
-//	_, err = newIpfs.Add(testDataBz)
-//	require.NoError(t, err)
-//
-//}
-//
-//func TestIpfsGet(t *testing.T) {
-//
-//	newIpfs := store.NewIpfs(ipfsNodeAddr)
-//
-//	file, err := ioutil.ReadFile("./testdata/test_deal.json")
-//	require.NoError(t, err)
-//
-//	cid, err := newIpfs.Add(file)
-//	require.NoError(t, err)
-//
-//	getStrings, err := newIpfs.Get(cid)
-//	require.NoError(t, err)
-//
-//	var deal types.Deal
-//	err = json.Unmarshal(file, &deal)
-//	require.NoError(t, err)
-//
-//	require.Equal(t, deal.DataSchema, getStrings)
-//}
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/medibloc/panacea-core/v2/x/datadeal/types"
+	"github.com/medibloc/panacea-doracle/ipfs"
+	"github.com/medibloc/panacea-doracle/ipfs/suite"
+	"github.com/stretchr/testify/require"
+)
+
+type ipfsTestSuite struct {
+	suite.TestSuiteIpfs
+}
+
+func TestIpfs(t *testing.T) {
+	suite.Run(t, new(ipfsTestSuite))
+}
+
+type testdata struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+}
+
+func (suite *ipfsTestSuite) TestIpfsAdd() {
+	newIpfs := ipfs.NewIpfs(suite.IpfsEndPoint(5001))
+
+	testData := &testdata{
```

Comment:
I think it would be good to change the value in testData.
Because if you add the same data, ipfs will always response the same cid.

Reply:
Does it mean use a data such as random number thing?

Reply:
Yes!